kbase.yml

```diff
@@ -8,7 +8,7 @@ service-language:
     python
 
 module-version:
-    1.2.3
+    1.2.4
```

Comment:
I don't see any functional changes. No need to bump the version.

Reply:
The tests are now functional, which is basically a bugfix, so I believe updating the patchlevel is appropriate.

Reply:
👍